sdk/servicebus/service-bus/test/utils/aadUtils.ts

```diff
@@ -1,21 +1,10 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-import { EnvVarNames, getEnvVars } from "./envVarUtils";
-
-import { loginWithServicePrincipalSecret } from "@azure/ms-rest-nodeauth";
-
-const aadServiceBusAudience = "https://servicebus.azure.net/";
+import { EnvironmentCredential } from "@azure/identity";
 
 export async function getTokenCredentialsFromAAD() {
-  const env = getEnvVars();
-  const tokenCreds = await loginWithServicePrincipalSecret(
-    env[EnvVarNames.AAD_CLIENT_ID],
-    env[EnvVarNames.AAD_CLIENT_SECRET],
-    env[EnvVarNames.AAD_TENANT_ID],
-    {
-      tokenAudience: aadServiceBusAudience
-    }
-  );
-  return tokenCreds;
+  return new EnvironmentCredential();
 }
+
+export { EnvironmentCredential } from "@azure/identity";
```

Comment:
Why do we need to re-export the `EnvironmentCredential` this way? Can the test files that need it not import it directly from `@azure/identity`?

Reply:
It's because EnvironmentCredential and @azure/identity don't seem browser compatible. So needed to do this to import in the tests per browserification guidelines.

Related question/discussion at https://github.com/Azure/azure-sdk-for-js/pull/5436#issuecomment-574306134
Could you clarify on that?

Reply:
Based on testing previously, this was added to address rollup and build issues - but with latest updates looks like the problem doesn't appear anymore - updated to remove this and aadUtils.ts files altogether